Create 1-introduction.md & README.md (#40)
Vaibhavpratapsingh22/Courses: 5-dbms/1-introduction.md

5) **Network Databases**: It is the database that typically follows the network data model. Here, the representation of data is in the form of nodes connected via links between them. Unlike the hierarchical database, it allows each record to have multiple children and parent nodes to form a generalized graph structure.

<img src="https://miro.medium.com/max/2214/1*ZuDEekCV_ScCSLSS8rT17g.png" height="" width="">

6) **Object-oriented Databases**: The type of database that uses the object-based data model approach for storing data in the database system. The data is represented and stored as objects which are similar to the objects used in the object-oriented programming language.

<img src="https://www.oreilly.com/library/view/hands-on-big-data/9781788620901/assets/593c262a-15ee-4277-91ea-1eb60be153a6.png" height="" width="">

7) **Hierarchical Databases**: It is the type of database that stores data in the form of parent-children relationship nodes. Here, it organizes data in a tree-like structure.

<img src="https://image2.slideserve.com/4935970/hierarchical-database-model-n.jpg" height="" width="">

> ## Importance and Applications

Reviewer: Applications are still missing here!

1. **Manages large amounts of data**: A database stores and manages a large amount of data on a daily basis. This would not be possible using any other tool such as a spreadsheet as they would simply not work.

2. **Easy to update data**: In a database, it is easy to update data using various Data Manipulation languages (DML) available. One of these languages is SQL.

3. **Security of data**: Databases have various methods to ensure the security of data. There are user logins required before accessing a database and various access specifiers. These allow only authorized users to access the database.

4. **Data integrity**: This is ensured in databases by using various constraints for data. Data integrity in databases makes sure that the data is accurate and consistent in a database.

5) **Cloud Database**: A type of database where data is stored in a virtual environment and executes over the cloud computing platform. It provides users with various cloud computing services (Software as a Service(SaaS), Platform as a Service(PaaS), Infrastructure as a service (IaaS), etc.) for accessing the database.

> 
> 
> 
> 

Reviewer: ```suggestion
```

# Chapter 1 - Introduction

> ## What is Database?

Reviewer: ```suggestion
```

5) **Network Databases**: It is the database that typically follows the network data model. Here, the representation of data is in the form of nodes connected via links between them. Unlike the hierarchical database, it allows each record to have multiple children and parent nodes to form a generalized graph structure.

<img src="https://miro.medium.com/max/2214/1*ZuDEekCV_ScCSLSS8rT17g.png" height="" width="">

6) **Object-oriented Databases**: The type of database that uses the object-based data model approach for storing data in the database system. The data is represented and stored as objects which are similar to the objects used in the object-oriented programming language.

<img src="https://www.oreilly.com/library/view/hands-on-big-data/9781788620901/assets/593c262a-15ee-4277-91ea-1eb60be153a6.png" height="" width="">

7) **Hierarchical Databases**: It is the type of database that stores data in the form of parent-children relationship nodes. Here, it organizes data in a tree-like structure.

<img src="https://image2.slideserve.com/4935970/hierarchical-database-model-n.jpg" height="" width="">

> ### Importance of Database:

Reviewer: ```suggestion
## Importance and Applications
```

Reviewer: and as I added to the title. Applications are missing here. Only importance is been given.

> The various reasons a database is important are −

Reviewer: ```suggestion
```

Reviewer: This is same thing repeated as the title. No repetitions.

2. **Data Updation**: It is used for the insertion, modification, and deletion of the actual data in the database.
3. **Data Retrieval**: It is used to retrieve the data from the database which can be used by applications for various purposes.
4. **User Administration**: It is used for registering and monitoring users, maintain data integrity, enforcing data security, dealing with concurrency control, monitoring performance and recovering information corrupted by unexpected failure.

### Advantages of DBMS

1. **Controls database redundancy**: It can control data redundancy because it stores all the data in one single database file and that recorded data is placed in the database.
2. **Data sharing**: In DBMS, the authorized users of an organization can share the data among multiple users.
3. **Easily Maintenance**: It can be easily maintainable due to the centralized nature of the database system.
4. **Reduce time**: It reduces development time.
5. **Backup**: It provides backup and recovery subsystems which create automatic backup of data from hardware and software failures and restores the data if required.

## Summary

> In this chapter we have discussed about database, its types and importance of database. Then we have a brief introduction to Database Management System(DBMS) and its advantages.

Reviewer: ```suggestion
In this chapter, we have discussed the database, its types and importance of database. Then we had a brief introduction to Database Management System(DBMS) and its advantages.
```
